docs/btc-methods.md

```diff
@@ -0,0 +1,73 @@
+# BTC Methods
+
+Here we document the Bitcoin methods that an account Snap may implement to
+support requests originated from dapps.
+
+## btc_sendmany
+
+This method is similar to the `sendmany` RPC method from Bitcoin Core, but its
+parameters are passed in an object instead of an array, and are named in
+camelCase. Also, dummy values aren't allowed.
+
+### Parameters
+
+- **Request object (required):**
+  - Type: `object`
+  - Properties:
+    - `amounts`
+      - Description: A JSON object with recipient addresses and amounts.
+      - Type: `object`
+      - Properties:
+        - `[key]: string`: Address of the recipient
+        - `[value]: string`: Amount to send to the recipient in BTC
+    - `comment` (optional)
```

Comment:
This PR only defines the API, how it will be used by the Snap is out of scope here.

In the Snap you can ignore this field if you want, or can display it in the confirmation dialog. I think it's not defined in the original spec how it should be used.

Reply:
IMO it's only useful for "tracking purposes", like we would see this `comment` for any requests being submitted to the keyring API, so this gives more "context" about the transactions.

To me, it shouldn't have any more meaning outside of the "request's scope" (the API implementer could use it for it own purposes)

Reply:
I'm going to resolve this since it's up to the Snap to decide how it's going to use the comment field.